### ISSUE TITLE
dice server down?

### ISSUE THREAD

**Issue by Numetalfan** [triplea-game/triplea#8329]
our play-by-email game worked fine,
but since yesterday, when trying to resolve the battle via dice calculator,
there is an error message, then the triplea.exe ist crashing and has to be restarted

Italians roll dice for 2 armour and 2 infantry in Belorussia, round 2
Contacting  dice.tripleawarclub.org
An error has occured!
Possible reasons the error could have happened:
  1: An invalid e-mail address
  2: Firewall could be blocking TripleA from connecting to the Dice Server
  3: The e-mail address does not exist
  4: An unknown error, please see the error console and consult the forums for help
     Visit https://forums.triplea-game.org/  for extra help
java.net.UnknownHostException: dice.tripleawarclub.org
	at java.net.Inet6AddressImpl.lookupAllHostAddr(Native Method)
	at java.net.InetAddress$2.lookupAllHostAddr(Unknown Source)
	at java.net.InetAddress.getAddressesFromNameService(Unknown Source)
	at java.net.InetAddress.getAllByName0(Unknown Source)
	at java.net.InetAddress.getAllByName(Unknown Source)
	at java.net.InetAddress.getAllByName(Unknown Source)
	at org.apache.http.impl.conn.SystemDefaultDnsResolver.resolve(SystemDefaultDnsResolver.java:45)
	at org.apache.http.impl.conn.DefaultHttpClientConnectionOperator.connect(DefaultHttpClientConnectionOperator.java:112)
	at org.apache.http.impl.conn.PoolingHttpClientConnectionManager.connect(PoolingHttpClientConnectionManager.java:373)
	at org.apache.http.impl.execchain.MainClientExec.establishRoute(MainClientExec.java:381)
	at org.apache.http.impl.execchain.MainClientExec.execute(MainClientExec.java:237)
	at org.apache.http.impl.execchain.ProtocolExec.execute(ProtocolExec.java:185)
	at org.apache.http.impl.execchain.RetryExec.execute(RetryExec.java:89)
	at org.apache.http.impl.execchain.RedirectExec.execute(RedirectExec.java:111)
	at org.apache.http.impl.client.InternalHttpClient.doExecute(InternalHttpClient.java:185)
	at org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:118)
	at games.strategy.engine.random.PropertiesDiceRoller.postRequest(PropertiesDiceRoller.java:147)
	at games.strategy.engine.random.PbemDiceRoller$HttpDiceRollerDialog.rollInSeperateThread(PbemDiceRoller.java:222)
	at java.lang.Thread.run(Unknown Source)

we never had any problems in the game before, no dice roll errors, no problems with a firewall or invalid email-addresses.
We are already in round 4, so alot of moves had been made in this game without error messages.

**Comment by DanVanAtta** [comment]
The domain expired. The URL is baked into the game version, if you load a game version that is more recent than July 2020 you'll pick up the updated domain. If you are capable to add a local DNS entry, that is a hack that could get you by until you can upgrade. I presume you are on version 1.9? If on version 2.0, you can just upgrade to the latest TripleA version and continue your game. If on 1.9, I'd recommend simply playing without the dice roller & finish your game.

**Comment by DanVanAtta** [comment]
For reference, this was the commit where the domain was updated: 78f15621d6a76c5900b3dd2bf38a1bc0af4f94b6

**Comment by RoiEXLab** [comment]
TL;DR Newer versions of TripleA don't have this problem, and manual workarounds are possible.
